Resources/Prototypes/SimpleStation14/Entities/Objects/Devices/Syndicate_Gadgets/sentient_gorilla_cube.yml

```diff
@@ -0,0 +1,22 @@
+- type: entity
+  parent: BaseItem
+  id: SentientGorillaCube
+  name: Sentient Gorilla Cube
+  description: A tiny teleportation device. After activation, once a syndicate-brand sentient gorilla is ready to take on the mission of assisting you, they will be teleported in to the device's location.
+  components:
+  - type: Sprite
+    sprite: SimpleStation14/Objects/Devices/gorillacube.rsi
+    layers:
+    - state: gorillacube
+    netsync: false
+  - type: GhostRole
+    name: Syndicate Gorilla
+    description: Someone needs reinforcements. You, the first gorilla the syndicate could find and upgrade, will help them.
```

Comment:
```suggestion
    description: Someone needs reinforcements. You, the first gorilla the syndicate could find, upgrade, and dehydrate will help them.
```
🥺?

Reply:
yeah whatever I'll just make it that and have the gorilla be a ghost role

Reply:
but see like the thing is is that I dont want the gorrila to spawn unless it can be useful

Reply:
I dont want the agent to just have an indefinitely AI controlled gorilla

Reply:
I think it's fine to leave as is. Maybe it just takes a while to hydrate. You could give it an active sprite for when it's awaiting a mind, like it sweating

Reply:
oh yea! A sprite change for when it's activated